### ISSUE TITLE
[Notification] Links as children do not meet AAA colour contrast compliancy

### ISSUE THREAD

**Issue by theetrain** [telusdigital/tds-core#582]
## Description

When using the Notification component and nesting the TDS Link component, the Link's text colour is a light grey that does not meet AAA WCAG colour contrast standards.

## Reproduction Steps

1. Go to https://tds.telus.com/components/index.html#!/Notification/1
2. Replace the existing example with the following:
```jsx
<Notification variant="branded">
  <Text bold>Tip:</Text> The services are best suited for larger business organizations ordering
  more than 50 plans on one account. <Link href="http://telus.com">Go to TELUS</Link>
</Notification>
```
3. Inspect the Link component and note the colour contrast 

## Reproduction Percentage (if applicable)

* 10/10 times

## Meta

* TDS version: all supported versions
* Willing to develop solution: Yes
* Has workaround: No
* High impact: No

## Screenshots

![image](https://user-images.githubusercontent.com/12798751/38742472-a2850f5e-3f0a-11e8-8da1-ff8d27d62037.png)

![image](https://user-images.githubusercontent.com/12798751/38742423-7e82e52c-3f0a-11e8-9b95-44f44683dbaf.png)

**Comment by ryanoglesby08** [comment]
Per design direction, the fix will be to change the color of all links (not including chevron links) to be the same as body text, "color-shark".

**Comment by marcod1419** [comment]
Fixed in this release: https://github.com/telusdigital/tds-core/releases/tag/%40tds%2Fshared-form-field%401.0.4
